ldk/csharp/OliveHelpsLDK/OliveHelpsLDK/Whispers/Forms.cs

```diff
@@ -4,22 +4,54 @@
 // ReSharper disable once CheckNamespace
 namespace OliveHelpsLDK.Whispers.Forms
 {
-    
-    public interface IWhisperFormResponse {}
+    /// <summary>
+    /// Base class for Whisper Form responses. Implementations will either be <see cref="WhisperUpdate"/> if
+    /// the user has updated a value, or <see cref="WhisperResult"/> when the form has been submitted.
+    /// </summary>
+    public interface IWhisperFormResponse
+    {
+    }
 
+    /// <summary>
+    /// Represents an incremental field update. 
+    /// </summary>
     public struct WhisperUpdate : IWhisperFormResponse
     {
+        /// <summary>
+        /// The key of the field updated.
+        /// </summary>
         public string Key;
+
+        /// <summary>
+        /// The output value of the field updated.
+        /// </summary>
         public IBase Output;
     }
 
+    /// <summary>
+    /// Represents the final submission of the form.
+    /// </summary>
     public struct WhisperResult : IWhisperFormResponse
     {
+        /// <summary>
+        /// Whether the form was submitted (<c>true</c>), or rejected/dismissed (<c>false</c>).
+        /// TODO: Confirm that dismissed forms return false.
```

Comment:
Has this been confirmed?

Reply:
There is no handling for dismissed whispers. I've opened SIDE-1355 to define it.